src/platform/i386/chal/shared/cos_config.h

```diff
@@ -164,6 +164,15 @@
                             /* 30/1G   31/2G */ \
                                -1,     -1 \
 
+/* PTE user pagetable modifiable flags */
+#define COS_PAGE_READABLE (0)
+#define COS_PAGE_WRITABLE (1ul << 1)
+#define COS_PAGE_PKEY0    (1ul << 59)
```

Comment:
A note for future github searches: we likely don't want to use these values in most code. Doing so will make the corresponding code non-portable.